global_costmap:
   global_frame: map
   map_topic: map
   robot_base_frame: base_footprint
   update_frequency: 0.3
   publish_frequency: 0.3
   static_map: true
   rolling_window: false
   resolution: 0.05
   transform_tolerance: 1.0
   map_type: costmap
   max_obstacle_height: 1.8
   min_obstacle_height: 0.0
   inflation_layer: 
      inflation_radius: 0.32
   obstacle_layer:
     observation_sources: scan  
     scan: {data_type: LaserScan, topic: /scan, marking: true, clearing: true, expected_update_rate: 15} 
   static_layer:
     lethal_cost_threshold: 1
   plugin:
     - {name: static_layer, type: "costmap_2d::StaticLayer"} 
     - {name: obstacle_layer, type: "costmap_2d::VoxelLayer"}
     - {name: inflation_layer, type: "costmap_2d::InflationLayer"}   #inflation_layer must be defined after obstacle_layer